packages/uicore/src/components/Thumbnail/Thumbnail.tsx

```diff
@@ -71,15 +71,19 @@ export const Thumbnail: React.FC<ThumbnailProps> = props => {
         className={css.squareCard}>
         {cardContent}
       </Card>
-      {(icon || imageProps?.src) && label && (
-        <Text
-          className={css.label}
-          font={{ weight: 'semi-bold' }}
-          color={disabled ? Color.GREY_500 : Color.GREY_600}
-          margin={{ top: 'small' }}>
-          {label}
-        </Text>
-      )}
+      {(icon || imageProps?.src) &&
+        label &&
+        (typeof label === 'string' ? (
```

Comment:
Why is this required?

Reply:
If someone wants to use label as a ReactElement, then we don't need to wrap it around the Text component & instead of using typeof to check for ReactElement existence, we can use isValidElement.